database/migrations/2021_05_01_155323_add_soc_to_readings.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
```

Comment:
Yo Lance, these are far too many migrations files that can be merged into 1 migration file. Instead of having each file to add a new column to the readings table, instead merge them all into a single migration file for the readings column, It should look something like 



Schema::table('readings', function (Blueprint $table) { 
           $table->longText('soc');  
           $table->string('rpm');     
       

 });